plasmapy/physics/parameters.py

```diff
@@ -394,23 +394,113 @@ def thermal_speed(T, particle="e", method="most_probable"):
     try:
         m = ion_mass(particle)
     except Exception:
-        raise ValueError("Unable to find {} mass in thermal_speed"
-                         .format(particle))
+        raise ValueError("Unable to find {particle} mass in thermal_speed")
 
     # different methods, as per https://en.wikipedia.org/wiki/Thermal_velocity
     if method == "most_probable":
-        V = (np.sqrt(2*k_B*T/m)).to(units.m/units.s)
+        V = (np.sqrt(2 * k_B * T / m)).to(units.m / units.s)
     elif method == "rms":
-        V = (np.sqrt(3*k_B*T/m)).to(units.m/units.s)
+        V = (np.sqrt(3 * k_B * T / m)).to(units.m / units.s)
     elif method == "mean_magnitude":
-        V = (np.sqrt(8*k_B*T/(m/np.pi))).to(units.m/units.s)
+        V = (np.sqrt(8 * k_B * T / (m / np.pi))).to(units.m / units.s)
     else:
-        raise(ValueError("Method {} not supported in thermal_speed"
-                         .format(method)))
+        raise ValueError("Method {method} not supported in thermal_speed")
 
     return V
 
 
+@check_relativistic
+@check_quantity({
+    'T': {'units': units.K, 'can_be_negative': False}
+})
+def kappa_thermal_speed(T, kappa, particle="e", method="most_probable"):
+    r"""Returns the most probable speed for a particle within a Kappa
+    distribution.
+
+    Parameters
+    ----------
+    T : Quantity
+        The particle temperature in either kelvin or energy per particle
+
+    kappa: Quantity
+        The kappa parameter is a dimensionless number which sets the slope
+        of the energy spectrum of suprathermal particles forming the tail
+        of the Kappa velocity distribution function. Kappa must be greater
+        than 3/2.
+
+    particle : string, optional
+        Representation of the particle species (e.g., 'p' for protons, 'D+'
+        for deuterium, or 'He-4 +1' for singly ionized helium-4),
+        which defaults to electrons.  If no charge state information is
+        provided, then the particles are assumed to be singly charged.
+
+    method : string, optional
+        Method to be used for calculating the thermal speed. Options are
+        'most_probable' (default), 'rms', and 'mean_magnitude'. 
+
+    Returns
+    -------
+    V : Quantity
+        particle thermal speed
+
+    Raises
+    ------
+    TypeError
+        The particle temperature is not a Quantity
+
+    UnitConversionError
+        If the particle temperature is not in units of temperature or
+        energy per particle
+
+    ValueError
+        The particle temperature is invalid or particle cannot be used to
+        identify an isotope or particle
+
+    UserWarning
+        If the particle thermal speed exceeds 10% of the speed of light, or
+        if units are not provided and SI units are assumed.
+
+    Notes
+    -----
+    The particle thermal speed is given by:
+
+    .. math::
+        V_{th,i} = \sqrt{(2 \kappa - 3)\frac{2 k_B T_i}{\kappa m_i}}
```

Comment:
This part definitely needs to show how the formula changes depending on the method chosen. I'm just reading the code for now, but I'll try to dig into reading up on that distribution soonish.

Reply:
I think this was mentioned in a different pull request, but do multi-line latex equations get properly rendered in the documentation?